index.bs

```diff
@@ -1938,31 +1886,59 @@ Issue: This needs to be generalized to work with realms too.
    </dd>
    <dt>Result Type</dt>
    <dd>
-      <pre class="cddl">
-      EmptyResult
+      <pre class="cddl local-cddl">
+        session.SubscriptionRequestResult = {
+          subscription: session.Subscription,
+        }
       </pre>
    </dd>
 </dl>
 
 <div algorithm="remote end steps for session.subscribe">
 The [=remote end steps=] with |session| and |command parameters| are:
 
-1. Let the |list of event names| be the value of the <code>events</code> field of
-   |command parameters|
+1. Let the |event names| be a [=/set=] containing |command parameters|["<code>events</code>"].
```

Comment:
This could be interpreted as a set containing a single list. The tedious way to do this would be something like
```
1. Let |event names| be a [=/set=].

1. For each |event| of |command parameters|["<code>events</code>"], [=set/append=] |event| to |event names|.
```

I think we could probably say `Let |event names| be |command parameters|["<code>events</code>"] interpreted as a [=/set=]`, but that's technically ambiguous if we have a duplicate element (infra just asserts that there are not duplicates in the set).

Reply:
Added a helper, will propose it to the infra spec.